test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperBlockRewriterTest.cs

```diff
@@ -675,12 +727,12 @@ public static TheoryData CodeTagHelperAttributesData
                                         "age",
                                         new MarkupBlock(
                                             new MarkupBlock(
-                                                factory.Markup("@").Accepts(AcceptedCharacters.None),
-                                                factory.Markup("@")
+                                                factory.CodeMarkup("@").Accepts(AcceptedCharacters.None),
+                                                factory.CodeMarkup("@")
                                                     .With(SpanChunkGenerator.Null)
                                                     .Accepts(AcceptedCharacters.None)),
                                             new MarkupBlock(
-                                                factory.EmptyHtml(),
+                                                factory.EmptyHtml(SpanKind.Code),
```

Comment:
Rather odd. Could this instead be `factory.EmptyCode()` and hide `private static Span EmptySpan(SpanFactory factory, SpanKind spanKind)` inside `TestSpanBuilder`?